src/Microsoft.Azure.WebJobs.Host/Config/ExtensionConfigContext.cs

```diff
@@ -63,12 +64,19 @@ public Uri GetWebhookHandler()
         /// <returns></returns>
         public FluentBindingRule<TAttribute> AddBindingRule<TAttribute>() where TAttribute : Attribute
         {
-            bool hasBindingAttr = typeof(TAttribute).GetCustomAttributes(typeof(BindingAttribute), false).Length > 0;
-            if (!hasBindingAttr)
+            var bindingAttrs = typeof(TAttribute).GetCustomAttributes(typeof(BindingAttribute), false);
+            if (!bindingAttrs.Any())
             {                
                 throw new InvalidOperationException($"Can't add a binding rule for '{typeof(TAttribute).Name}' since it is missing the a {typeof(BindingAttribute).Name}");
             }
 
+            var isTrigger = typeof(TAttribute).Name.EndsWith("TriggerAttribute", StringComparison.OrdinalIgnoreCase);
```

Comment:
>var isTrigger = typeof(TAttribute).Name.EndsWith("TriggerAttribute", StringComparison.OrdinalIgnoreCase); [](start = 12, length = 105)

Can you make this an instance method on the BindingAttribute? Makes it more discoverable.